Client/Frontend/Components/LabelButtonHeaderView.swift

```diff
@@ -17,6 +17,10 @@ struct LabelButtonHeaderViewModel {
     static var emptyHeader: LabelButtonHeaderViewModel {
         return LabelButtonHeaderViewModel(title: nil, isButtonHidden: true)
     }
+
+    static func == (lhs: LabelButtonHeaderViewModel, rhs: LabelButtonHeaderViewModel) -> Bool {
+        return lhs.title == rhs.title && lhs.isButtonHidden == rhs.isButtonHidden
+    }
 }
```

Comment:
Where possible I prefer to keep code that is exclusively used for tests contained to the test target. In your test file or in a seperate file you can extend LabelButtonHeaderViewModel and have it conform to equatable.